helpers/PalladiumHelper.rb

```diff
@@ -18,18 +18,33 @@ def get_result_set_link
     "#{@palladium.result_set_id}"
   end
 
-  def add_result_and_log(example)
-    result = add_result(example)
+  def add_result_and_log(example, file_data = nil)
+    result = add_result(example, file_data)
     OnlyofficeLoggerHelper.log("Test is #{result['status']['name']}")
     OnlyofficeLoggerHelper.log(get_result_set_link)
   end
 
-  def add_result(example)
+  def add_result(example, file_data = nil)
     name = example.metadata[:description]
-    status, comment = get_status(example)
+    status, comment = get_status_detailed_comment(example, file_data)
     @palladium.set_result(name: name, description: comment, status: status)
   end
 
+  # @@return [Array] adds additional information about the file size,
```

Comment:
This is not correct method description

Try to run 
```
yardoc helpers/PalladiumHelper.rb
```
and it show warning because of example format

And resulting documentation looks like: (note that @return statement not parsed and example not formatted properly)
![image](https://user-images.githubusercontent.com/668524/83440931-64934980-a44e-11ea-9777-0a0b72ea1ffe.png)

Better take a look at yard cheatsheet [here](https://gist.github.com/chetan/1827484)